ruby_on_rails/introduction/web_refresher.md

```diff
@@ -127,7 +132,7 @@ It's also how some ads seem to follow you from one website to another -- another
 
 On the server side, you'll interact with cookies and session variables quite a bit.  As mentioned above, one of the main uses of these is to determine who the user is, or "authentication".  You'll basically retrieve the cookie that the user sends you, use it to find that user in your database, and (if the user exists) then you can display the customized web page for that user.
 
-It's pretty straightforward in theory, but some of the security implications get a bit hairy so luckily the nice folks at [Platformatec](http://plataformatec.com.br/) created a very handy gem called ["Devise"](https://github.com/plataformatec/devise) which takes care of all this stuff for you.  In this curriculum (a bit later on), you'll be creating your own authentication system before learning how to use Devise to take care of the heavy lifting.
+It's pretty straightforward in theory, but some of the security implications get a bit hairy so luckily the nice folks at [Platformatec](http://plataformatec.com.br/) created a very handy gem called ["Devise"](https://github.com/heartcombo/devise) which takes care of all this stuff for you. _(__note:__ Since 2020, the gem is not maintained by Platformatec and the ownership has been given to contributors and maintainers of Devise. [See official blog post here](https://blog.plataformatec.com.br/2020/01/important-information-about-our-elixir-and-ruby-open-source-projects/) for more information.)_  In this curriculum (a bit later on), you'll be creating your own authentication system before learning how to use Devise to take care of the heavy lifting.
```

Comment:
Good idea to clarify this.

Personally I would strip out all reference to Platformtec altogether at this point. I'm not sure it adds anything particularly valuable.

Reply:
So would:
> but some of the security implications get a bit hairy so luckily some nice folks have created a very handy gem called ["Devise"](https://github.com/heartcombo/devise) which takes care of all this stuff for you.

Be sufficient here without all this Platformatec-change-of-hands thing?

Reply:
Perfect :ok_hand:

Reply:
Should be done now!